host-configs/docker/gcc@11.1.0.cmake

```diff
@@ -97,11 +97,13 @@ set(SUPERLUDIST_DIR "${TPL_ROOT}/superlu-dist-8.1.2-i66ldlr2l325korbopnlirbk53fz
 
 set(RAJA_DIR "${TPL_ROOT}/raja-2022.10.5-6hoezdnmcqqaghnhyqqfjq7kgtcbmxab" CACHE PATH "")
 
+# STRUMPACK not built
```

Comment:
Is strumpack not supposed to be built on blueos and docker containers? @jamiebramwell

Reply:
We build and test it on LC.